util/drakeGeometryUtil.cpp

```diff
@@ -812,6 +812,61 @@ GradientVar<typename DerivedI::Scalar, TWIST_SIZE, TWIST_SIZE> transformSpatialI
   return ret;
 }
 
+template<typename DerivedA, typename DerivedB>
+Eigen::Matrix<typename DerivedA::Scalar, TWIST_SIZE, DerivedB::ColsAtCompileTime> crossSpatialMotion(
```

Comment:
made these templated and put them in drakeGeometryUtil. WIll delete crm et al. at the top of RigidBodyManipulator once we switch to the new kinsol format.